packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -29,6 +27,11 @@ $unselectedHoverColor: $ms-color-neutralLighter;
   &.ms-DetailsRow.is-selected {
     background: $selectedColor;
   }
+
+  &.prevent-selection {
```

Comment:
I thought about it, but it felt weird to me to add and then immediately remove. `is-contentUnselectable` might work and be more in line with the other modifiers